src/middlewared/debian/control

```diff
@@ -156,6 +157,9 @@ Depends: alembic,
          python3-websocket,
          python3-zeroconf,
          python3-zettarepl,
+         # This is required by incus VMs
+         qemu-system-modules-spice,
```

Comment:
This module alone brings in 163 more packages....that's insane. Is there any way we can work around this?